content/authentication/troubleshooting-ssh/host-key-verification-failed.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Host key verification failed
+intro: 'As a security precaution, ssh keeps track of which hosts it has previously seen.'
+versions:
+  fpt: '*'
+  ghec: '*'
+topics:
+  - SSH
+shortTitle: Host key verification failed
+---
+
+This means that the key that the server to which your client is connecting presented a key that doesn't match the keys that your client has seen in the past.
```

Comment:
There's some duplication here

Is there a way we can use language that is slightly more beginner-friendly than 'client' here? For example, could we just say "the server to which you're connecting"? Alternatively, can we be slightly more explicit as to what we mean (e.g. "your SSH client")?

```suggestion
This error means that the server to which your client is connecting presented a key that doesn't match the keys that your client has seen in the past.
```

Reply:
So, `client` here on average, more or less, means `git`. In general that's effectively `git.exe` or `/usr/bin/git`, although if someone is using [`jgit`](https://git-scm.com/book/uz/v2/Appendix-B%3A-Embedding-Git-in-your-Applications-JGit), or [go-git](https://github.com/go-git/go-git) or [dulwich](https://github.com/jelmer/dulwich) (which is sometimes used, e.g. by Mercurial to talk to git via [HgGit](https://wiki.mercurial-scm.org/HgGit)), [libgit2](https://github.com/libgit2/libgit2), or [js-git](https://github.com/creationix/js-git), it could mean one of those.

But, it could mean any [ssh implementation](https://ssh-comparison.quendi.de/) in case someone is actually just connecting directly via `ssh`.

Reply:
Basically, I'm using `client` to cover for the fact that I don't know if they're using `git` (or an implementation) or `ssh` (or an implementation).

Reply:
And, if they're using `git` (or an implementation) or worse a wrapper for git, like an IDE -- whether that's IntelliJ, VSCode, Eclipse, or something else) -- I can't know that they'll really know what `ssh` is or that they're using it.

Reply:
I think "the server to which you're connecting" might be a useful abstraction here, but again go with your preference

Reply:
I'll take that, but it still leaves a `client` at the end of the sentence.

One might consider:
```diff
-doesn't match the keys that your client has seen in the past.
+doesn't match the keys that it has used in the past.
```

But this could easily be a globally incorrect statement while technically accurate from the perspective of the local client. And I'm not terribly comfortable w/ that tradeoff. (Can you tell I like my precision?) 

But, let's try a variant of that and see what you think.